source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,165 @@
+You can use the [GOV.UK skip link component from the design system](https://design-system.service.gov.uk/components/skip-link/)
+for this.
 
-## Main content
+### Sectioned content
 
-The `<main>` tag should be used to identify the main content of your page. Support for it is still marginal so ensure to add the `role="main"` attribute.
+Use the `<section>` tag to represent standalone, thematic groupings of content
+where a more specific semantic element would not be appropriate.
 
-## Sectioned content
+Examples of sections would be chapters, or various tabbed pages in a tabbed dialog box,
+or a home page split into sections such as introduction, news items and contact information.
 
-When using the `<section>` tag, add an `aria-labelledby` attribute linking to the id of the heading that identifies that section.
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
 
 Example:
 
-    <section aria-labelledby="advice">
-      <h1 id="advice">Help and advice</h1>
+```html
+<section aria-labelledby="introduction">
+    <h1 id="introduction">Introduction</h1>
+    <p>People have been catching fish for food since before recorded history...</p>
+</section>
 
-## Navigational elements
+<section aria-labelledby="equipment">
+    <h1 id="equipment">Equipment</h1>
+    <p>The first thing you'll need is a fishing rod or pole that you find comfortable
+    and is strong enough for the kind of fish you're expecting to try to land...</p>
+</section>
+```
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+### Navigational elements
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). Common examples of navigation sections are
+menus, tables of contents, and indexes.
 
-### Emphasis
+As with `<section>`, it is advisable to use `aria-labelledby` to label your `<nav>`.
+This helps to distinguish different `<nav>` blocks in the page.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+The GOV.UK blog has [more information regarding the use of the `<nav>` tag](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
 
-Example:
+### Aside
+
+`<aside role="complementary">` should be used for content related to the primary content
+of the webpage, but which does not constitute the primary content of the page.
+
+Examples include author information, related links and related content.
+
+### Footer
+
+`<footer role="contentinfo">` should be used for the footer of the site.
+
+## Individual element guidance
+
+### Headings
+
+Headings should be in order - for example `<h1>` then `<h2>`, but not `<h1>` then `<h3>`.
+
+There should only be one `<h1>` element in a page.
+
+### Text emphasis
```

Comment:
I appreciate that you took the time to rewrite this section but I think it's unnecessary. I'm sorry, I should have been clearer about that from the start.

Reply:
Thanks for looking. I think there's value in having this section, since this document is about HTML markup and it's so common to see em/strong/b/i misused. Worth a discussion in the next frontend meeting?

Reply:
In meeting:

- don't use italics - not supported by our font
- screen reader spec says items read with emphasis, but not implemented
- link to content style guide for GOV.UK
- @selfthinker to review guide

Reply:
Have now pushed up the changes.

Reply:
I have reviewed everything, left some comments, some about accessibility, some not. But none of what I raised is a blocker. :+1: